ios/AppDelegate.swift

```diff
@@ -91,41 +96,35 @@ class AppDelegate: RCTAppDelegate {
       self.overview.alpha = 1
     })
   }
-  
+
   func hideOverview() {
     guard let view = self.window.viewWithTag(OVERVIEW_TAG) else {
       return
     }
-    
+
     UIView.animate(withDuration: OVERVIEW_FADE_DURATION, animations: {
       view.alpha = 0
     }) { _ in
       view.removeFromSuperview()
     }
   }
-  
-  @objc
-  func concurrentRootEnabled() -> Bool {
```

Comment:
This was an unused method?

Reply:
@devkudasov https://react-native-community.github.io/upgrade-helper/?from=0.71.12&to=0.72.4#RnDiffApp-ios-RnDiffApp-AppDelegate.mm

Reply:
> @devkudasov https://react-native-community.github.io/upgrade-helper/?from=0.71.12&to=0.72.4#RnDiffApp-ios-RnDiffApp-AppDelegate.mm

@ragozin-nikita FYI